docs/docs/Support/release-notes.mdx

```diff
@@ -91,7 +91,10 @@ For all changes, see the [Changelog](https://github.com/langflow-ai/langflow/rel
 
   New [**CometAPI** bundle](/bundles-cometapi)
 
-  New [**Docling Remote VLM** component](/bundles-docling#docling-remote-vlm)
+- Knowledge base components
```

Comment:
Once you have all the subbranches merged into 1.7-docs, give the release notes another look to make sure they are cohesive.